db/migrate/20220210132254_remove_checkbox_parent_fields.rb

```diff
@@ -11,7 +11,7 @@ def down
     change_table :case_logs, bulk: true do |t|
       t.column :accessibility_requirements, :string
       t.column :condition_effects, :string
-      t.remove :reasonable_preference_reason, :string
+      t.column :reasonable_preference_reason, :string
```

Comment:
Why is this coming back?

Reply:
This is for `down` method. I ran a rollback and it failed because of this methinks (?)

Reply:
Sorry stupid question